exasol_integration_test_docker_environment/lib/base/base_task.py

```diff
@@ -170,6 +170,11 @@ def get_log_path(self) -> Path:
         path.mkdir(parents=True, exist_ok=True)
         return path
 
+    def get_main_log_path(self) -> Path:
```

Comment:
I don't know, why you want to have this part of the task. However, if you still want it here, you should   take a look at _get_output_path_for_job, they share parts.

Reply:
You were faster with reviewing than I with developing ;)
In the final version this was removed.

Reply:
It is still there

Reply:
Now it's fixed